rpm/busybox-sailfish.config

```diff
@@ -989,6 +996,7 @@ CONFIG_UDHCPC_DEFAULT_SCRIPT="/usr/share/udhcpc/default.script"
 #
 # Common options for DHCP applets
 #
+CONFIG_UDHCPC_DEFAULT_INTERFACE="eth0"
```

Comment:
I'm not quite sure which value this should have. This is the default that `make oldconfig` sets.